src/components/ContributeDetails.js

```diff
@@ -0,0 +1,111 @@
+import PropTypes from 'prop-types';
```

Comment:
As stated in https://github.com/opencollective/opencollective-frontend/pull/1196#discussion_r243185801 either we disable ESLint `react/react-in-jsx-scope` or we import React. As of today this generates a warning